docs/conf.py

```diff
@@ -0,0 +1,124 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=invalid-name
+
+"""Sphinx documentation builder."""
+
+# -- General configuration ---------------------------------------------------
+import datetime
+import doctest
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+sys.path.append(os.path.abspath("."))
```

Comment:
I had to add this fix for sphinx to find the path to the modules. Without it, I got `attempted relative import beyond top-level package`. If there is anything else that can be done to fix this, let me know.